audit/audit.go

```diff
@@ -88,42 +88,59 @@ func outputCSV(loud bool, projects []types.Coordinate) (int, string) {
 					vv.Cve,
 					vv.CvssVector,
 				})
+				if err != nil {
+					return
+				}
 			}
 		} else {
-			w.Write([]string{
+			err = w.Write([]string{
 				v.Coordinates,
 				v.Reference,
 			})
+			if err != nil {
+				return
+			}
 		}
+		return
 	}
 
 	b := new(bytes.Buffer)
 	w := csv.NewWriter(b)
 
 	if loud {
 		for _, v := range projects {
-			writer(w, v)
+			err := writer(w, v)
+			if err != nil {
+				return 0, "", err
+			}
 		}
 		w.Flush()
-		return len(vulnerablePackages), b.String()
+		return len(vulnerablePackages), b.String(), nil
 	}
 
 	for _, v := range vulnerablePackages {
-		writer(w, v)
+		err := writer(w, v)
+		if err != nil {
+			return 0, "", err
+		}
 	}
 	w.Flush()
-	return len(vulnerablePackages), b.String()
+	return len(vulnerablePackages), b.String(), nil
 }
 
-func outputText(noColor bool, loud bool, projects []types.Coordinate) (int, string) {
+func outputText(noColor bool, loud bool, projects []types.Coordinate) (int, string, error) {
 	var sb strings.Builder
 
 	w := tabwriter.NewWriter(&sb, 9, 3, 0, '\t', 0)
 	w.Flush()
```

Comment:
minor: and not even part of the PR, but since it's on my mind...: maybe quite IDE warnings by adding a: `_ = yadda`?

`_ = w.Flush()`

Reply:
@bhamail Fixed in 1e81398